fbw-a32nx/src/systems/instruments/src/EFB/Failures/FailureGenerators/PerHourFailureGeneratorUI.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2021-2023 FlyByWire Simulations
+//
+// SPDX-License-Identifier: GPL-3.0
+
+import { usePersistentProperty } from '@flybywiresim/fbw-sdk';
+
+import React, { useMemo, useState } from 'react';
+import { FailureGenContext, FailureGenData, setNewSetting } from 'instruments/src/EFB/Failures/FailureGenerators/RandomFailureGenEFB';
+import { t } from 'instruments/src/EFB/translation';
+import { FailureGeneratorSingleSetting, FailureGeneratorText } from 'instruments/src/EFB/Failures/FailureGenerators/FailureGeneratorSettingsUI';
+
+const settingName = 'EFB_FAILURE_GENERATOR_SETTING_PERHOUR';
+const additionalSetting = [3, 1, 2, 0.1];
```

Comment:
What meaning do those numbers have?

Reply:
In general, the array of default settings for all generators start with the 

- mode 
0=OFF
1=ONCE
2=Take Off
3 = Repeat <-- this one by default for this generator
- Number of failures to trigger at once (1 by default for this generator)
- max number of failures at once allowed to trigger this generator (2 by default for this generator)

Then, all the subsequent numbers of the array are specific to the generator.
The PerHour gen 's 4th number if the probability per hour 0.1/hour = 1 failure every 10 hours by default

Reply:
Can you represent these as ENUMs instead?

Reply:
Done, created enums and variables sor that it describes what it does on all gens

Reply:
Looks much better, thanks!